scripts/extract-paired-reads.py

```diff
@@ -48,9 +60,22 @@ def get_parser():
     parser = argparse.ArgumentParser(
         description='Take a mixture of reads and split into pairs and '
         'orphans.', epilog=textwrap.dedent(epilog))
-    parser.add_argument('infile')
+    parser.add_argument('infile', nargs='?', default='/dev/stdin')
     parser.add_argument('--version', action='version', version='%(prog)s ' +
                         khmer.__version__)
+
+    parser.add_argument('-o', '--output-dir',
+                        dest='output_directory', default='', help='Output '
```

Comment:
On Sun, Jun 14, 2015 at 12:40:25PM -0700, Michael R. Crusoe wrote:
> >      parser.add_argument('--version', action='version', version='%(prog)s ' +
> >                          khmer.__version__)
> > +
> > +    parser.add_argument('-o', '--output-dir',
> > +                        dest='output_directory', default='', help='Output '
> 
> Thanks for removing the `metavars`. `dest` is still a bit redundant: one can either rename the option to `--output-directory` or change the references to be `args.output_dir` and the `dest` specification can be removed.

ok. I don't think this should hold up merges, frankly.